features/nationaliteithistorie.feature

```diff
@@ -3,128 +3,441 @@
-    | 2      | 54        |       |       | 403   | 20170526 |
-    | 2      | 54        |       |       | 402   | 20170210 |
-    | 2      | 54        | 0449  | 301   |       | 19930419 |
-
-  En de ingeschreven persoon met burgerservicenummer 999992466 kent de volgende nationaliteiten:
-    | Stapel | Categorie | 05.10 | 63.10 | 64.10 | 85.10    | 84.10 |
-    | 1      | 04        | 0057  | 301   |       | 20160601 |       |
-    | 1      | 54        | 0057  | 301   |       | 20160501 |       |
-    | 1      | 54        | 0057  | 301   |       | 20160401 |       |
-    | 1      | 54        | 0057  | 301   |       | 20160301 | O     |
-
-  En de ingeschreven persoon met burgerservicenummer 999992855 kent de volgende nationaliteiten:
-    | Stapel | Categorie | 05.10 | 63.10 | 64.10 | 85.10    | 84.10 |
-    | 1      | 04        | 0001  | 018   |       | 20041015 |       |
-    | 2      | 04        |       |       |       | 20121022 |       |
-    | 2      | 54        | 0251  |       |       | 20121022 | O     |
-    | 3      | 04        | 0445  |       |       | 19611230 |       |
-    | 4      | 04        | 0446  |       |       | 19611230 |       |
-
-  Scenario: actuele en beëindigde nationaliteiten in de juiste volgorde
-    Als de nationaliteithistorie met burgerservicenummer 999991292 wordt geraadpleegd
-    Dan bevat het antwoord 3 voorkomens
-    En wordt de nationaliteithistorie teruggegeven in de volgorde en met waarden:
-    | # | nationaliteit.code | aanduidingBijzonderNederlanderschap | datumIngangGeldigheid | datumTot             | indicatieNationaliteitBeeindigd | redenBeeindigen.code |
-    | 0 | 0001               | -                                   | 1991-02-01            | -                    | -                               | -                    |
-    | 0 | 0057               | -                                   | 1983-12-13            | 2014-06-01           | true                            | 404                  |
-    | 0 | 0100               | -                                   | 1989-03-01            | 1994-06-01           | true                            | 404                  |
-
-  Scenario: meerdere actuele en beëindigde nationaliteiten
-    Als de nationalteithistorie met burgerservicenummer 999990457 wordt geraadpleegd
-    Dan bevat het antwoord 5 voorkomens
-    En wordt de nationaliteithistorie teruggegeven in de volgorde en met waarden:
+  # in onderstaande scenario's staat ten behoeve van de provider implementatie in Gegeven achter de veldnaam tussen haakjes de veldcode zoals deze in de bron volgens LO GBA is geregistreerd
+
+  Rule: In het antwoord wordt indicatieNationaliteitBeeindigd opgenomen met de waarde true, wanneer in de actuele nationaliteit (categorie 04) GEEN nationaliteit (05.10) noch aanduiding bijzonder Nederlanderschap (65.10) is opgenomen, of wanneer in categorie 04 reden beëindigen nationaliteit (64.10) is opgenomen.
+
+    Scenario: actuele nationaliteit
```

Comment:
'actuele nationaliteit heeft een waarde' als titel zou volgens mij deze scenario beter beschrijven

Reply:
Ik probeer de titel van het scenario (zoveel mogelijk) te beschrijven zoals het in de werkelijkheid bestaat. ik wijzig het naar "vastgestelde nationaliteit". 
Bijvoorbeeld scenario "actuele en beëindigde nationaliteit" heb ik hernoemd naar "beëindigde registratie van vreemde nationaliteit", wat verwijst naar de reden beëindigen.

Is deze benadering onduidelijk en zou je de titel(s) ook vanuit de registratie willen benoemen?

Reply:
Volgens mij wil je de mapping beschrijven van de PL naar de HC BRP. Wat ik uit de scenario concludeer is dat als 05.10 een waarde heeft, dat deze terug te vinden is in nationaliteit.code. Vandaar mijn voorstel om de titel daarop aan te passen

Reply:
> Volgens mij wil je de mapping beschrijven van de PL naar de HC BRP

ja, maar ik wil ook beschrijven hoe situaties die in de werkelijkheid voorkomen in de API terecht komen. Dat kan helpen voor gebruikers, maar dat helpt ons (mij) ook om te zien of de API in deze situaties op de gewenste manier werkt.

Ik heb de titel aangepast naar "de persoon heeft de nationaliteit nu"